src/Sulu/Component/Security/Authorization/SecurityChecker.php

```diff
@@ -39,9 +39,8 @@ public function __construct(
 
     public function hasPermission($subject, $permission)
     {
-        if (!$subject || !$this->tokenStorage->getToken()) {
```

Comment:
`tokenStorage->getToken()` returned an `AnonymousToken` if no user was logged in previously. This was changed in the recently introduced authenticator based security model: https://symfony.com.ua/doc/current/security/experimental_authenticators.html

With the new security model, `tokenStorage->getToken()` will return `null` if no user is logged in.